src/Document/GarbageCollection.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Content
+ *
+ * @package Wp_Graphql_Smart_Cache
+ */
+
+namespace WPGraphQL\SmartCache\Document;
+
+use WPGraphQL\SmartCache\Admin\Settings;
+use WPGraphQL\SmartCache\Document;
+use GraphQL\Server\RequestError;
+
+class GarbageCollection {
+
+	const TAXONOMY_NAME = 'graphql_document_skip_gc';
```

Comment:
Originally used the 'skip' word to describe per document query.  Noticed left this here. Should I change to 'disable_gc' to match rest of language?